src/app/core/core.env.ts

```diff
@@ -7,6 +7,7 @@ const DEFAULTS = {
   CMS_TTL: 1, // 1 second
   CMS_ROOT_TTL: 3600, // 1 hour
   CMS_USE_LOCALSTORAGE: false,
+  FEEDER_HOST: 'https://feeder.prx.org',
```

Comment:
Does Publish need to actually know where Feeder is?  CMS is already configured to know, and returns absolute links to Feeder (or any other PRX Distribution Service) in the `distribution` / `story_distribution` representers.  The idea being that Publish can re-use the same ID auth token to authenticate against any PRX service, and they're all HAL, so we can navigate across them seamlessly.